WordPress/Classes/ViewRelated/Support/SupportChatBot/SupportChatBotViewController.swift

```diff
@@ -189,15 +189,16 @@ extension SupportChatBotViewController {
     func createTicket(with history: SupportChatHistory) {
         SVProgressHUD.show(withStatus: Strings.ticketCreationLoadingMessage)
 
-        viewModel.contactSupport(including: history, in: self) { [weak self] success in
+        viewModel.contactSupport(including: history, in: self) { [weak self] result in
             SVProgressHUD.dismiss()
 
             guard let self else { return }
             DispatchQueue.main.async {
-                if success {
-                    self.viewModel.track(.supportChatbotTicketSuccess)
+                switch result {
+                case let .success(request):
+                    self.viewModel.track(.supportChatbotTicketSuccess, ticketNumber: request.requestId)
```

Comment:
Have you tested what `request.requestId` returns? I'm not sure if it returns the correct id. I couldn't retrieve the actual ticket number, and I believe this is how it's intended in the Zendesk API. Since we're creating the request from the user side, the user can't be aware of the ticket number, as it's company-specific data.

Reply:
You're right, `requestId` is returning an alphanumeric ID instead of the numeric ticket ID we see in Zendesk. I'll keep looking.

Reply:
> I believe this is how it's intended in the Zendesk API. Since we're creating the request from the user side, the user can't be aware of the ticket number, as it's company-specific data.

That's a good point.